src/vm/loaderallocator.cpp

```diff
@@ -546,7 +546,8 @@ BOOL QCALLTYPE LoaderAllocator::Destroy(QCall::LoaderAllocatorHandle pLoaderAllo
 
     BEGIN_QCALL;
 
-    if (ObjectHandleIsNull(pLoaderAllocator->GetLoaderAllocatorObjectHandle()))
+    OBJECTHANDLE objHandle = pLoaderAllocator->GetLoaderAllocatorObjectHandle();
+    if (*(Object**)objHandle == NULL)
```

Comment:
No reason... I just didn't think it was valuable since the `== NULL` suffices. It was also used in just two places and in two asserts so it wasn't a big deal to modify the callers.

I don't have a particularly strong opinion on this, so if you feel strongly I can add `ObjectHandleIsNull` to gchandletableutilities.